.changeset/config.json

```diff
@@ -10,13 +10,13 @@
   "baseBranch": "main",
   "updateInternalDependencies": "patch",
   "ignore": [
-    "amplify-ui-angular-mono",
```

Comment:
This mind also need to be changed? [└── package.json # name: 'amplify-ui-angular-mono`, private: true](https://github.com/aws-amplify/amplify-ui/blob/b9d19c49621e5cfd0da06d5cff5ea42c092abed0/packages/angular/CONTRIBUTING.md#L23-L24)

Reply:
Thanks! I removed these references in https://github.com/aws-amplify/amplify-ui/pull/3811/commits/a18b61963d4a1c3565cbd6af8699add66db0bb53.